app/admin/controllers/Payments.php

```diff
@@ -152,7 +152,7 @@ public function formValidate($model, $form)
             'status' => ['required', 'integer'],
         ];
 
-        $messages = [
+        $attributes = [
```

Comment:
Yup I wanted to make this change, but they were so consistently called messages I figured you must have had a reason!